index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="media-style.css">
+    <title>Anicet's Capstone Project</title>
+</head>
+
+<body>
+    <header>
+        <div class="logo"><a href=""><img src="./images/NFTSbck.jpg" alt="This a NFT logo"></a></div>
+
+        <ul>
+            <li><a href="About.html">About</a></li>
+            <li><a href="">Progran</a></li>
+            <li><a href="">Join</a></li>
+            <li><a href="">Sponsors</a></li>
+            <li><a href="">News</a></li>
+            <li><a href="">CC campaign</a></li>
+        </ul>
+    </header>
+    <main>
+        <div class="first-part">
+            <div class="nav-btn">
+                <div class="menu-line"></div>
+                <div class="menu-line"></div>
+                <div class="menu-line"></div>
+            </div>
+
+            <ul class="menu-bar">
+                <li class="close-button">&times;</li>
+                <li><a href="About.html">About</a></li>
+                <li>Progran</li>
+                <li>Join</li>
+                <li>Sponsors</li>
+                <li>News</li>
+                <li>CC campaign</li>
+            </ul>
+
+            <p class="punchline">"Hello cryptography world"</p>
+            <h1>Start the the day with coffee. <br> End the day with NFTS</h1>
+            <p class="punchline2">Anyone who follows the crypto space knows that non-fungible tokens (NFTs) are all the
+                rage of late.</p>
+            <div class="for-date">2021.05.11(THU) ~ 05(FRI)</div>
+            <div class="endline">@ National crypto of Democratic Republic of Congo</div>
+        </div>
+
+        <div class="for-cards">
+            <h2>Main Programs</h2>
+            <div class="card-container">
+                <div class="card00">
+                    <div class="icon"><img src="./images/time-money.png" alt=""></div>
+                    <div class="card-content">
+                        <h3>time-money</h3>
+                        <p>Reach out our marketplace, learn more on how to invest in time and make more experience with
+                            us </p>
+                    </div>
+                </div>
```

Comment:
- [ ] Please kindly transform the `<div class="card00">` cards into a more semantic element like an `<article>` to make the code cleaner and easier to navigate and read.